UniWatch/Views/Class/StudentIndex.cshtml

```diff
@@ -0,0 +1,57 @@
+﻿@model IEnumerable<UniWatch.Models.Class>
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h3>My Classes</h3>
+
+@if (!Model.Any())
+{
+    @:It's empty in here. Create a new class.
```

Comment:
Copy and paste? A student cannot create a class. Change to something like "It's empty in here. Your classes will show up once a teacher enrolls you in his/her class"